recipes/sentry-breakpad/all/conanfile.py

```diff
@@ -0,0 +1,152 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.28.0"
+
+
+class SentryBreakpadConan(ConanFile):
+    name = "sentry-breakpad"
+    description = "Client component that implements a crash-reporting system."
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/getsentry/breakpad"
+    license = "Apache-2.0"
+    topics = ("conan", "breakpad", "error-reporting", "crash-reporting")
+    provides = "breakpad"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "fPIC": True,
+    }
+    exports_sources = "CMakeLists.txt", "patches/*"
+    generators = "cmake"
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def requirements(self):
+        if self.settings.os in ("FreeBSD", "Linux"):
+            self.requires("linux-syscall-support/cci.20200813")
+
+    def validate(self):
+        if self.settings.compiler.cppstd:
+            tools.check_min_cppstd(self, 11)
+
+        if tools.Version(self.version) <= "0.4.1":
+            if self.settings.os == "Android" or tools.is_apple_os(self.settings.os):
+                raise ConanInvalidConfiguration("Versions <=0.4.1 do not support Apple or Android")
+        if tools.Version(self.version) <= "0.2.6":
+            if self.settings.os == "Windows":
+                raise ConanInvalidConfiguration("Versions <=0.2.6 do not support Windows")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], destination=self._source_subfolder)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.configure()
+        return self._cmake
+
+    def _patch_sources(self):
+        for patch in self.conan_data.get("patches", {}).get(self.version, []):
+            tools.patch(**patch)
+
+        # FIXME: convert to patches
```

Comment:
Is this FIXME on the scope of this PR? Or it will be addressed in the future?

Reply:
I waited to convert them until I got it building successfully on c3i.
I will convert them asap.